packages/itwin/tree-widget/src/components/trees/repositories-tree/RepositoriesService.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * @internal
+ */
+export interface ITwinRepositories {
+  repositories: ITwinRepository[];
+}
+
+interface Repository {
+  id?: string;
+  displayName?: string;
+  type?: string;
+  name?: string;
+}
+
+interface ITwinRepository {
+  id?: string;
+  class: string;
+  subClass?: string;
+  uri?: string;
+}
+
+/**
+ * @internal
+ */
+export async function getItwinRepositories(itwinId: string, accessToken: string, environment?: "PROD" | "QA" | "DEV"): Promise<ITwinRepository[]> {
+  const url = getRepositoriesUrl(itwinId, environment);
+  const result = await fetchData<ITwinRepository[]>(url, accessToken);
+  return result;
+}
+
+/**
+ * @internal
+ */
+export async function getRepositoryData(accessToken: string, url?: string): Promise<Repository[]> {
```

Comment:
I believe this function returns not repositories, but their content, doesn't it? Return type should be named accordingly.

Reply:
changed name